staging/src/k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -185,6 +194,8 @@ message ContainerAllocateResponse {
 	repeated DeviceSpec devices = 3;
 	// Container annotations to pass to the container runtime
 	map<string, string> annotations = 4;
+	// CDI devices for the container.
+	repeated CDIDevice cdi_devices = 5;
```

Comment:
Let's define this as follows so that the go code interprets this as CDIDevices, not CdiDevices:
```
cdi_devices = 5 [(gogoproto.customname) = "CDIDevices"];
```

Reply:
OK. I can update. For what it's worth, I was just following what was done for the CRI spec, but I prefer the capitilized version and will update.